### DISCUSSION TITLE
RTL support

### DISCUSSION THREAD

**Discussion by hlageek** [RE-QDA/requal#146] [Show and tell]
I am leaving the issue here as a reminder to ourselves after noticing the problem of RTL being raised recurrently in discussions about CAQDAS. Implementing this would increase reQual's inclusiveness as well as utility for many branches in the humanities. Unfortunately, this will be non-trivial, as we will have to solve several problems, the main ones being:
- how to represent RTL text in DB
- how to count offsets in RTL text
- how to adjust displayed RTL text in UI
And these problems are interconnected 😰

**Comment by martin-hajek** [comment]
And what about using a browser RTL extension? 
![image](https://github.com/RE-QDA/requal/assets/54745940/3990977c-6736-4be2-9300-74944f2021c8)

**Comment by hlageek** [comment]
> And what about using a browser RTL extension? 

That is a great tip to share with RTL users 👏 Not sure if the text rendering/coding will be correctly handled by this extension. Nevertheless, I discussed it with colleagues at TCDH and we should be able to implement some support on at least a basic level without much difficulty. All thanks to the modern browser accessibility features. Going the `shiny` way again comes to the rescue ⭐️
